koko-analytics.php

```diff
@@ -1,16 +1,14 @@
 <?php
 /*
 Plugin Name: Koko Analytics
-Plugin URI: https://dvk.co/#utm_source=wp-plugin&utm_medium=koko-analytics&utm_campaign=plugins-page
 Version: 1.0
+Plugin URI: https://dvk.co/#utm_source=wp-plugin&utm_medium=koko-analytics&utm_campaign=plugins-page
 Description: Privacy-friendly analytics for your WordPress site.
 Author: ibericode
 Author URI: https://ibericode.com/#utm_source=wp-plugin&utm_medium=koko-analytics&utm_campaign=plugins-page
-Author Email: support@kokoanalytics.com
 Text Domain: koko-analytics
 Domain Path: /languages/
-License: GPL-3.0
-License URI: https://www.gnu.org/licenses/gpl-3.0.txt
+License: GPL v3
```

Comment:
Looks like some (newer) commits were accidentally reverted, can you change the PR to include the latest commit (or at least not revert it)?

Reply:
Agh, so sorry. I am quite new on doing pull requests from a fork, I will fix both problems.

Reply:
@dannyvankooten I applied your recent changes to my newest commits. Sorry about that!